frontend/components/App.jsx

```diff
@@ -40,7 +40,7 @@ export default function App() {
   //     "altitudeAccuracy":-1
   //   },
   //   "timestamp":1669174971110.204
-//   }
+  //   }
 
   let text = 'Waiting...';
   if (errorMsg) {
```

Comment:
Maybe this was from the previous PR logging whether the user granted location permission. Looks like it's only logging when there's a geolocation but not when there's an error message. Move console.log outside the else if.